doc/installation.rst

```diff
@@ -34,25 +38,35 @@ Just visit the `Spyder page on Binder`_ to get started using Spyder now.
 
 
 
-.. _standalone_installers_ref:
+.. _install-standalone:
 
 =====================
 Standalone installers
 =====================
 
-Our standalone installers for Windows and macOS are available from Spyder 4.2 onwards.
-We recommend using this installation method on those platforms, but we offer several other options for Linux, advanced users and specific needs, so keep reading if that's the case for you.
+The standalone installers are our recommended method for most users on Windows and macOS, with experimental Linux support under active development.
+They work much like installing any other IDE, where Spyder can be installed and updated independently of the Python environments you use to run your code, avoiding problems with incompatible packages and broken installations that users often face when doing so within their own Python installation.
+
+The installers include a built-in Python environment with the most common scientific libraries, which can be used out of the box for basic data analysis tasks.
```

Comment:
```suggestion
The installers include a built-in Python environment with the most common scientific libraries (e.g. Matplotlib Pandas and Numpy), which can be used out of the box for basic data analysis tasks.
```

Just so users get a feeling of what they're going to find in the installers.

Reply:
```suggestion
The installers include a built-in Python environment with the most common scientific libraries (e.g. NumPy, Pandas, Matpotlib, etc), which can be used out of the box for basic data analysis tasks.
```

Fix punctuation, capitalization and ordering